taxi/views.py

```diff
@@ -1,3 +1,28 @@
+import json
+import os
+
+from django.http import HttpRequest, HttpResponse
 from django.shortcuts import render
 
-# Create your views here.
+from collections import Counter
+
+
+def index(request: HttpRequest) -> HttpResponse:
+    base_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    file_path = os.path.join(base_dir, "taxi_service_db_data.json")
+    if os.path.exists(file_path):
+        with open(file_path, "r") as file:
+            data = json.load(file)
+
+        models_counter = Counter(item["model"] for item in data)
+        num_drivers = models_counter["taxi.driver"]
+        num_manufacturers = models_counter["taxi.manufacturer"]
+        num_cars = models_counter["taxi.car"]
```

Comment:
This is a bad solution. You should retrieve the data count in the database using Django ORM. Here is an example: `Driver.objects.count()`.